sdk/core/core-util/test/public/datetime.spec.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { toOffsetDateTime } from "../../src/datetime";
+import { assert } from "chai";
+import moment from "moment";
+
+describe("utcDateTime with timezone offset", function () {
+  describe("work at current datetime", function () {
+    it("the unixtime diff should be correct in default milliseconds precision", function () {
+      const date = new Date();
+      assert.strictEqual(
+        Math.floor(
+          Math.abs(Date.parse(toOffsetDateTime(date)) - Date.parse(date.toISOString())) / 1000
```

Comment:
Here as the toOffsetDateTime will lost the miliiseconds of three digit, and we have no way to assert the real diff, so I think we can assume the quotient of the diff divided by 1000 is zero.